src/content/docs/about/getting-started.mdx

```diff
@@ -0,0 +1,273 @@
+
+Daytona supports a wide range of Git Providers, including popular platforms like [GitHub](/configuration/git-providers#github), [GitLab](/configuration/git-providers#gitlab), and [Bitbucket](/configuration/git-providers#bitbucket), as well as self-hosted solutions. Integrating a Git Provider with Daytona allows you to manage your repositories directly within the platform, eliminating the need for repeated authentication.
+
+<br />
+
+<CardGrid gridNumber={3} mobileGridNumber={1} colsMinWidth='150px' colsMinWidthMobile='100px'>
+    <TitleCard icon={githubLogo} title="GitHub" link="/configuration/git-providers#github" imageAlt="'GitHub logo"/>
+    <TitleCard icon={gitlabLogo} title="GitLab" link="/configuration/git-providers#gitlab" imageAlt="GitLab logo"/>
+    <TitleCard icon={bitbucketLogo} title="Bitbucket" link="/configuration/git-providers#bitbucket" imageAlt="BitBucket logo"/>
+</CardGrid>
+
+For more details on available Git Providers and related commands, use the `daytona git-providers` command to explore options and manage your configurations.
+
+<DocumentList>
+<DocumentListItem
+    title="Add a Git Provider"
+    subtitle="Learn how to add a Git Provider to Daytona and start managing your repositories effectively."
+    href="/configuration/git-providers"
+    expanded
+  />
+</DocumentList>
+
+## Installing a Provider
+
+[Provider](/configuration/providers) is a plugin that interfaces with the [Daytona Server](/usage/server), handling development environment deployment and lifecycle management.
+
+Daytona supports a variety of Providers, allowing you to integrate with different platforms based on your needs. Once installed, a Provider will manage the deployment of your development environment directly within Daytona, ensuring integration with your chosen platform.
+
+For more details on available Providers and related commands, use the `daytona provider` command to explore options and manage your configurations.
+
+<DocumentList>
+<DocumentListItem
+    title="Install a Provider"
+    subtitle="Learn how to install a Provider and interface with the Daytona Server."
+    href="/configuration/git-providers"
+    expanded
+  />
+</DocumentList>
+
+## Setting a Target
+
+Once you have installed a Provider, you can set a [Target](/configuration/providers#add-a-target) to define how Daytona manages and deploys your development environment. Configuring a Target allows you to tailor the development environment for your projects.
+
+A Target is a configuration that specifies where and how your development environment will be deployed. It is associated with a Provider and includes all the details Daytona needs to create and manage your development environment in a specific location, such as a local machine, cloud server, or remote Docker instance.
+
+Once your Target is set, Daytona will use it whenever you create a new development environment. You can manage multiple Targets, switch between them, or delete those you no longer need.
```

Comment:
`switch between them`

There is no switching between them. You choose a target when creating a workspace and that's it. It's essentially just a workspace option that can not be changed.